app/controllers/identify_controller.rb

```diff
@@ -111,11 +116,14 @@ def log_certificate(cert)
       valid_policies: cert.valid_policies?,
       valid: validation_result == 'valid',
       error: validation_result != 'valid' ? validation_result : nil,
+      openssl_valid: status.success? && stderr.ends_with?("OK") && errors.empty?,
+      openssl_errors: errors.join(', '),
     }
 
     if validation_result == 'self-signed cert'
       attributes.delete(:issuer)
     end
+    puts attributes
```

Comment:
just gotta make sure we remove this before merging

Reply:
Yep, thanks for catching that, removed.